index.js

```diff
@@ -17,219 +24,240 @@ const TOKEN_PATH = 'token.json';
 
 const dismoji = require('discord-emoji');
 const enm = require('emoji-name-map');
+
 const EMOJI_ONE = enm.get('one');
-const EMOJI_TWO = enm.get('two');
-const EMOJI_BOOM = enm.get('boom')
+// const EMOJI_TWO = enm.get('two');
+const EMOJI_BOOM = enm.get('boom');
 
 const BOT_STATE_REF = 'Dashboard!B4';
 
 const REFS = {
-  'round': 'Dashboard!B2',
-  'song': 'Dashboard!B3',
-  'header': 'Dashboard!D1',
-  'footer': 'Dashboard!D8',
-  'match': 'Dashboard!D3:E6',
-  'size': 'Dashboard!B5',
-  'year': 'Dashboard!B1'
-}
+  round: 'Dashboard!B2',
+  song: 'Dashboard!B3',
+  header: 'Dashboard!D1',
+  footer: 'Dashboard!D8',
+  match: 'Dashboard!D3:E6',
+  size: 'Dashboard!B5',
+  year: 'Dashboard!B1',
+};
 
 let skipstat = false;
 let testing = false;
 let once = false;
 
-process.argv.forEach(function (val, index, array) {
-    if( val === '-s' ) { skipstat = true;}
-    if( val === '-t' ) { testing = true;}
-    if( val === '-o' ) {once = true;}
-  });
+process.argv.forEach((val) => {
+  if (val === '-s') { skipstat = true; }
+  if (val === '-t') { testing = true; }
+  if (val === '-o') { once = true; }
+});
 
-const GUILD_ID = (testing === true ? '782213860337647636' : '212660788786102272');  
-const CHANNEL_ID =  (testing === true ? '864768873270345788' : '751893730117812225');  
+const GUILD_ID = (testing === true ? '782213860337647636' : '212660788786102272');
+const CHANNEL_ID = (testing === true ? '864768873270345788' : '751893730117812225');
 const SPREADSHEET_ID = (testing === true ? '1-xVpzfIVr76dSuJO8SO-Im55WQZd0F07IQNt-hhu_po' : '1qQBxqku14GTL70o7rpLEQXil1ghXEHff7Qolhu0XrMs');
 
-
-const START_TIME = (skipstat === true ? 0 : 5)
-const END_TIME = (skipstat === true ? 24 : 21)
+const START_TIME = (skipstat === true ? 0 : 5);
+const END_TIME = (skipstat === true ? 24 : 21);
 
 const now = new Date();
 
-const isBotEnabled = botState => {
-  let nowHour = new Date().getHours();
+const isBotEnabled = (botState) => {
+  const nowHour = new Date().getHours();
   return botState === 'GO' && START_TIME < nowHour && nowHour < END_TIME;
-}
-
-
-const formatMatchRow = row => '\u200b' + row[0] + '\u200b' + ((typeof row[1] != 'undefined') ? ` ${row[1]}` : '');
+};
 
+const formatMatchRow = (row) => `\u200b${row[0].trim()}\u200b${(typeof row[1] !== 'undefined') ? ` ${row[1]}` : ''}`;
+const formatOther = (text) => text.replaceAll(/:[^:\n]+:/g, '\u200b$1\u200b');
 
-const getMatchText = rows => rows.map(formatMatchRow).join('\n');
+const getMatchText = (rows) => rows.map(formatMatchRow).join('\n');
 
-
-const getDismojiByName = name => {
-  for (let cat of Object.values(dismoji)) {
-    if (typeof cat[name] != 'undefined') {
+const getDismojiByName = (name) => {
+  const namedDismoji = Object.values(dismoji).map((cat) => {
```

Comment:
This will not behave the same as the old function and probably introduced a bug. Instead of returning a single emoji it would return an array with a `null` value for each category except for the category (categories?) that have matching emoji. I would leave it as it was before, though the `let` can be a `const`:
```
const getDismojiByName = (name) => {
  for (const cat of Object.values(dismoji)) {
    if (typeof cat[name] !== 'undefined') {
      return cat[name];
    }
  }
  return null;
};
```

Reply:
Okay. Reverting.